tensorflow/lite/micro/micro_graph.cc

```diff
@@ -162,10 +166,16 @@ TfLiteStatus MicroGraph::InvokeSubgraph(int subgraph_idx) {
 // This ifdef is needed (even though ScopedMicroProfiler itself is a no-op with
 // -DTF_LITE_STRIP_ERROR_STRINGS) because the function OpNameFromRegistration is
 // only defined for builds with the error strings.
-#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+#ifndef TF_LITE_DISABLE_PROFILING
```

Comment:
we needed 2 additional flags anyway, so it seemed preferable to use the ones that make more sense, semantically.